### PR TITLE
Support API 4.0.0

### PR DESCRIPTION
**Pull request by NhanAZ**

### CODE REVIEW

File: src/KygekDev/AntiBedrock/AntiBedrock.php [hunk 1]
```diff
@@ -14,14 +14,14 @@
...
-use pocketmine\block\Block;
+use pocketmine\block\VanillaBlocks;
 use pocketmine\event\block\BlockBreakEvent;
 use pocketmine\event\Listener;
 use pocketmine\plugin\PluginBase;
 
 class AntiBedrock extends PluginBase implements Listener {
 
-    public function onEnable() {
+    public function onEnable() : void {
```
Comment (KygekDev) [inline src/KygekDev/AntiBedrock/AntiBedrock.php]: Make it similar with `PluginBase`:

```suggestion
    protected function onEnable() : void {
```

**Comment by KygekDev** [comment]
Thanks for your contribution. Before I merge your PR, may I know if you have tested your changes or not?

**Comment by NhanAZ** [comment]
> Thanks for your contribution. Before I merge your PR, may I know if you have tested your changes or not?

Tested and it works! Hope so...

**Comment by KygekDev** [comment]
> > Thanks for your contribution. Before I merge your PR, may I know if you have tested your changes or not?
> 
> Tested and it works! Hope so...

@NhanAZ Thanks, I will merge this PR now

**Pull request merged**